vcpkg-vendor/fix_vendor_libs.py

```diff
@@ -152,6 +152,7 @@
         "/usr/lib/libSystem.B.dylib",
         "/usr/lib/libc++.1.dylib",
         "/usr/lib/libcharset.1.dylib",
+        "/usr/lib/libffi.dylib",
```

Comment:
we _build_ libffi via vcpkg, we definitely shouldn't be using the system libffi

https://github.com/koordinates/kart/blob/ea74d9ea0cf86cce3e2adf75ee298e157fcf57af/vcpkg-vendor/vcpkg.json#L73